.gitignore

```diff
@@ -27,3 +27,5 @@ yarn-error.log*
 config
 .env
 test
+
+package-lock.json
```

Comment:
Usually, you would always want to commit the package.json. Are you ignoring it because of the huge amount of conflicts coming from it? You should resolve them and then commit the change.

Reply:
Oh, no real reason, I just thought I was best practice to not include it.  I will remove it from the gitignore.